src/init.js

```diff
@@ -0,0 +1,75 @@
+/*------------------------------------------------------------------------------------------------\
+  Initialize adverse event explorer.
+\------------------------------------------------------------------------------------------------*/
+
+export function init(data) {
+    var element = d3.select(this.element);
+    var settings = this.config;
+
+  //Render single column if no group variable is specified.
+    if (!(settings.variables.group) || ['', ' '].indexOf(settings.variables.group) > -1) {
+        settings.variables.group = 'data_all';
+        settings.defaults.totalCol = '';
+        settings.groups = [{'key':'All'}];
+    }
+
+    function errorNote(msg) {
```

Comment:
Lets move the settings checks to a separate method in a follow up release. Created #33.